2fa-read-code/src/index.js

```diff
@@ -19,20 +19,40 @@ const lookBackMinutes = process.env.look_back_minutes;
       }
       const captcha = readCaptchaFromMessage(msg);
       if (captcha) {
-        const subject = readSubjectFromMessage(msg);
-        res.push(
-          utils.buildItem({
-            title: `${captcha}`,
-            subtitle: `Sender：${subject} ${dateUtils.formatToCalendar(
-              messageObj.message_date
-            )}，⏎ to Copy`,
-            arg: captcha,
-            text: {
-              largetype: messageObj.text,
-              copy: captcha
-            }
-          })
-        );
+        subject = readSubjectFromMessage(msg);
+        if (!subject) {
+          // `subject` is falsey, which includes `undefined` and `null`
+          // (and `""`, and `0`, and `NaN`, and [of course] `false`)
+          // https://stackoverflow.com/a/5101991
+          res.push(
+            utils.buildItem({
```

Comment:
Too much duplicate code, I gave up on merging. I will fix it directly based on this issue. thanks again.